web/client/components/misc/FeatureInfoTriggerSelector.jsx

```diff
@@ -17,14 +17,22 @@ class FeatureInfoTriggerSelector extends React.Component {
     static propTypes = {
         trigger: PropTypes.string,
         onSetMapTrigger: PropTypes.func,
+        onPurgeMapInfoResults: PropTypes.func,
```

Comment:
this is not related to the pr, i guess you need to start creating a branch from a brand new branch **after** you have update your local master branch